content/docs/command-reference/plots/index.md

```diff
@@ -1,20 +1,20 @@
 # plots
 
-Contains commands to visualize _plot metrics_ in structured files (JSON, CSV, or
-TSV): [show](/doc/command-reference/plots/show),
-[diff](/doc/command-reference/plots/diff).
+A set of commands to visualize and compare _plot metrics_ in structured files
+(JSON, YAML, CSV, or TSV): [show](/doc/command-reference/plots/show),
+[diff](/doc/command-reference/plots/diff), and
+[modify](/doc/command-reference/plots/modify).
 
 ## Synopsis
 
 ```usage
-usage: dvc plots [-h] [-q | -v] {show,diff} ...
+usage: dvc plots [-h] [-q | -v] {show,diff,modify} ...
 
 positional arguments:
   COMMAND
-    show         Generate a plot image file from a metrics file.
-    diff         Plot differences in metrics between commits in the
-                 DVC repository, or between the last commit and the
-                 workspace.
+    show         Generate plot from a metrics file.
+    diff         Plot differences in metrics between commits.
+    modify       Modify plot props associated with a target file.
```

Comment:
props -> settings? properties?

Reply:
It is already `properties` in core dvc. Fixed earlier today. This is probably just a leftover.

Reply:
We had a mix of "props" and "properties" in the docs since it was just `props` in core.

Reply:
Extracted this to #1414 for now since it seems it's in active development still.

Reply:
OK this was just a grep/replace so I did it in 1bcae8d7